### ISSUE TITLE
Login Gitblit using Azure AD users

### ISSUE THREAD

**Issue by flexruleresources** [gitblit/gitblit#1394]
I know there is option to install Azure Domain services (using secure Ldaps) and then connect/sync users by adding realm.authenticationProviders = ldap options in the gitbilt.property;
My question is, can I authentication users with AD Azure directly without going through above process? or has anyone done this in different way?

**Comment by flaix** [comment]
I don't have an answer to that. But I would like to ask for clarification what "with AD Azure directly" would mean? Do you mean with your Azure account, or do you mean with an Azure Active Directory? If the latter, what would "directly" mean, are there examples for this? I guess it means employing Kerberos for the authentication?

**Comment by flexruleresources** [comment]
You are right, its Azure Active Directory; e.g. I add service account user and give it required permission (read) to be setup on Gitblit.
"directly" probably is not right word, but I meant not to install Domain service on Azure.

**Comment by flaix** [comment]
But isn't an Azure AD also just an AD? So the same principles would apply.
